.github/workflows/build_and_test.yml

```diff
@@ -147,10 +150,71 @@ jobs:
           mvn -B -T1C clean install
         shell: cmd
 
+        # This step needs a Git repository, so it's impossible to extract it
+        # into a separate job (or, otherwise, we'd need to upload the content
+        # of the whole `.git` folder as an artifact).
+      - name: JUnit Tests (dorny/test-reporter@v1)
+        uses: dorny/test-reporter@v1
```

Comment:
What is the difference between this action and `publish-unit-test-result-action`? As I can see, this one creates a report on worklfow's summary, but the latter seems to do this too?